lib/java/ydoc-server/src/main/java/org/enso/ydoc/server/Main.java

```diff
@@ -16,21 +21,40 @@ public static void main(String[] args) throws Exception {
         "helidon.serialFilter.pattern",
         "javax.management.**;java.lang.**;java.rmi.**;javax.security.auth.Subject;!*");
 
-    var ydocHost = System.getenv(ENSO_YDOC_HOST);
-    var ydocPort = System.getenv(ENSO_YDOC_PORT);
-
-    var builder = Ydoc.builder();
-    if (ydocHost != null) {
-      builder.hostname(ydocHost);
-    }
-    if (ydocPort != null) {
-      var port = Integer.parseInt(ydocPort);
-      builder.port(port);
+    if (args.length == 2) {
+      var then = System.currentTimeMillis();
+      var hostname = args[0];
+      var port = args[1];
+      launch(hostname, port);
+
+      var now = System.currentTimeMillis();
+      var took = now - then;
+      log.warn("Ydoc server at {}:{} started in {} ms", hostname, port, took);
```

Comment:
```suggestion
      log.debug("Ydoc server at {}:{} started in {} ms", hostname, port, took);
```

Warnings are always seen by users. There is no need for that here.